### PR TITLE
Feature/module actions

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Add actions to be executed at the end of a portal module
# Impacts

Portal modules
# Delete branch after merge

YES
# NEWS file entries
## New Features
- Added configurable actions to be executed at the end of a portal module

### CODE REVIEW

**Comment by julsemaan** [comment]
ready to be reviewed

**Comment by julsemaan** [comment]
Good for review

File: html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Module.pm [hunk 1]
```diff
@@ -38,6 +39,33 @@ has username => (is => 'rw', builder => '_build_username', lazy => 1);
 
 has renderer => (is => 'rw');
 
+has 'actions' => ('is' => 'rw', isa => 'HashRef', default => sub {{}});
+
+sub BUILD {
```
Comment (jrouzierinverse) [inline html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Module.pm:50]: documentation

**Comment by jrouzierinverse** [comment]
Looks good will merge after documentation is updated

**Comment by julsemaan** [comment]
doc added

**Pull request merged**